doc/how_to/index.rst

```diff
@@ -1,13 +1,14 @@
-How to guides
-=============
+How to Guides
+=========
+
+Guides on how to solve specific, short problems in spike interface. Learn how to...
```

Comment:
```suggestion
Guides on how to solve specific, short problems in SpikeInterface. Learn how to...
```
or it could be spikeinterface. Just in case you haven't seen the difference between the two. The overall project is stylized SpikeInterface whereas the package specifically is stylized as spikeinterface.